__tests__/app.test.js

```diff
@@ -97,4 +104,50 @@ describe("GET /api/articles/:article_id", () => {
     })
   })
 })
+describe("GET /api/articles", () => {
+  test("200: responds with array of article objects each with specific properties, and without body property", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({body})=>{
+      const {articles} = body
+      if (articles.length){
+        articles.forEach((article)=>{
+          expect(article).toMatchObject(
+            {
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              article_img_url: expect.any(String),
+              article_id: expect.any(Number),
+              comment_count: expect.any(String),
+              votes: expect.any(Number)
+            }
+          )
+        })
+      }
+    })
+  })
+  test("200: responds with array of article objects in ascending order of date created", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({body})=>{
+      const {articles} = body
+      expect(articles).toBeSortedBy("created_at")
+    })
+  })
```

Comment:
I like this test but the task asks for the date to be in **_descending_** order, not ascending, it may be worth going back to this and refactoring your test and function.